docker-compose/docker-compose.yaml

```diff
@@ -1,5 +1,56 @@
 version: '2.4'
 services:
+  # Description: This container will migrate the Postgres `frontend`,
+  # `codeintel`, and `codeinsights` databases
+  #
+  # Disk: None
+  # Ports: None
+  #
+  # This container should start after the database instance starts, perform the migrations
+  # and then exit cleanly. After exiting the `frontend` service is allowed to start.
+  #
+  # If you're using a DB instance hosted outside of docker-compose, the environment variables
+  # for this container will need to be updated to reflect the new connection information.
+  migrator:
+    container_name: migrator
+    image: 'index.docker.io/sourcegraph/migrator:3.35.1'
+    cpus: 4
```

Comment:
The cpu/mem limit here seems really high for a migrator container - can we figure out a better benchmark?

Reply:
I copied these specs from the `frontend` containers with the expectation that once these migrations are done, the container terminates and should release the resources. We can lower them dramatically, the migrator needs very little resources to issue the SQL, the real lift is on the DB itself.